ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessService.scala

```diff
@@ -215,7 +217,7 @@ class DBProcessService(managerActor: ActorRef,
         }
         processUpdated <- EitherT(repositoryManager
           .runInTransaction(processRepository
-            .updateProcess(UpdateProcessAction(processIdWithName.id, substituted, action.comment, increaseVersionWhenJsonNotChanged = false))
+            .updateProcess(UpdateProcessAction(processIdWithName.id, substituted, UpdateProcessComment(action.comment), increaseVersionWhenJsonNotChanged = false))
```

Comment:
You can use `Comment` class in `UpdateProcessCommand` and than `UpdateProcessComment` will be created in real place of comment creation. Thanks to that it can have signature `UpdateProcessComment(environmentId, user)` and we will see in one place all possible system comments

Reply:
I'm not certain if i understand this one, I would like to discuss it further